grammars/ruby.cson

```diff
@@ -2285,6 +2310,35 @@
         'name': 'comment.line.number-sign.ruby'
       }
     ]
+  'separators':
+    'patterns': [
+      {
+        'match': '=>'
+        'name': 'punctuation.separator.key-value.ruby'
+      }
+      {
+        'match': ';'
+        'name': 'punctuation.separator.statement.ruby'
+      }
+      {
+        'match': ','
+        'name': 'punctuation.separator.object.ruby'
+      }
+      {
+        'match': '(::)\\s*(?=[A-Z])'
```

Comment:
The `\\s*` can be in the lookahead.

Reply:
For some reason I thought lookaheads can only have a fixed number of characters.